homeassistant/components/tankerkoenig/__init__.py

```diff
@@ -52,6 +53,8 @@
                 vol.Optional(CONF_STATIONS, default=[]): vol.All(
                     cv.ensure_list, [cv.string]
                 ),
+                vol.Optional(CONF_SHOW_ON_MAP, default=False
```

Comment:
Setting the default to false might break the functionality of anyone relying on the longitude and latitude attributes being available.